zip-0340.md

```diff
@@ -0,0 +1,537 @@
+can spend at the snapshot height. 
+
+The ZH is not obligated to vote with all the ZEC that they have.
+But it MUST be impossible to use the same coins to vote
+twice on the same election.
+
+It MUST be possible to use the same ZEC on DIFFERENT elections.
+
+**See Privacy section on important caveats regarding concurrent
+voting.**
+
+## Other Uses Cases
+
+### Airdrop
+This ZIP can be used to perform air-drops. An organization
+that wants to give their tokens/coins based on an amount
+of ZEC can organize an election. The ballots contain
+the ZH's address where they want to receive the airdrop.
+
+### Proof of Balance
+In this case, the ZH sends an empty message because the
+ballot value is all that matters.
+
+# Requirements
+
+## Participants / Actors
+
+### Election Board
+
+The Election Board (EB) is the entity in charge of organizing
+the election (E). 
+- The EB MUST have a website secured by https that clearly
+reflects their identity. 
+- They MUST make their announcements 
+through their website. 
+- The EB SHOULD have a forum for election related discussions
+- The EB MUST set the election parameters described in a later
+section
+- The EB MUST publish the election parameters on their website
+- The election parameters are immutable. If the EB
+wants to make a modification, another election MUST be created
+
+### Voter (ZH)
+
+Anyone who has *spending authority* of ZEC 
+(either transparent or shielded) can 
+participate.
+
+- Ballots MUST have a ZEC value
+- Ballots MUST have a message in the format defined by the Election
+- Ballots MUST include a proof of spending authority for the
+amount specified, valid at the snapshot height
+- Ballots MUST not prevent the ZH from spending the coins in a transaction
+- Coins spent before the snapshot height MUST not be eligible for voting
+- Voting MUST not disclose information that would allow someone 
+to use the coins themselves (in a transaction or an election)
+- Ballots MUST follow the format specified in the section Ballot
+- ZEC held in transparent t3 addresses CANNOT be used. They introduce
+most complexity and are not used much
+
+Notes:
+- Having coins on an exchange such as Binance, Bitfinex, etc.
+does not allow you to vote because the coins are under their custody
+- The EB MAY set special rules for the election. For example,
+there MAY be a fee for casting a vote or a minimum ZEC requirement
+
+## Privacy
+:::danger
+If the coins used for voting for a given election are not moved
+before voting for another election, it is possible to link
+the votes.
+:::
+
+1. Transparent inputs are in clear
+1. Shielded inputs reveal their nullifier
+
+The total value is also revealed since it is the weight
+of the vote.
+
+To mitigate this issue, the user SHOULD be offered to move
+their coins after a vote. However, if multiple elections
+are carried out concurrently with similar snapshot
+heights, it will be impossible for a Voter to participate
+in both without linking their votes.
```

Comment:
This is the one paragraph that should stay in the Specification section.

Also, conformance requirements should be specific about what parties they place requirements upon:
```suggestion
Wallets SHOULD offer to move their user's coins after a vote. However, if
multiple elections are carried out concurrently with similar snapshot heights,
it will be impossible for a Voter to participate in both without linking their
votes.
```